src/php/k8s-php-install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+ZEND_API_NO=$1 # zend representation of the PHP version number
+AGENT_FILEPATH=/newrelic-instrumentation/agent
+DAEMON_FILEPATH=/newrelic-instrumentation/daemon
+INSTALL_LOG=/newrelic-instrumentation/newrelic-install.log
+
+log() {
+  msg="$1"
+  timestamp=$(date +"%m-%d-%Y %T")
+  echo "${timestamp} : ${msg}" >>${INSTALL_LOG}
+}
+
+if [ -z ${ZEND_API_NO} ]; then
+  log "Missing Zend api number. Exiting."
+  exit 1
+fi
+
+log "Installing New Relic PHP Agent for the K8S Agent Operator"
+
+#
+# Determine the architecture we are executing on.
+#
+arch=$( (uname -m) 2>/dev/null) || arch="unknown"
+case "${arch}" in
+  aarch64 | arm64) arch=aarch64 ;;
+  *64* | *amd*) arch=x64 ;;
```

Comment:
I think this would match the Power PC 64 bit `uname -m` output of "ppc64" - could we make the first test more restrictive for x64 arches?

Reply:
this is lifted from the base `newrelic-install.sh` script, so it's probably a problem there as well.